contributing/release-schedule.md

```diff
@@ -1,84 +1,84 @@
 
-## Determining if your change is in production
+The _weekly_ release channels are considered to be the primary "evergreen" release channels. A green build from the `master` branch is used to build the **experimental** and **beta** release channels; the **beta** release from the previous week is promoted to the **stable** release channel (see the [Detailed schedule](#detailed-schedule)).
 
-You can determine what is in a given build using one of the following:
+There are two sets of build configurations used in creating release builds: the _canary_ configuration and the _production_ configuration. The **experimental** and **beta** release channels are built off of the same commit, but with the _canary_ and _production_ configurations, respectively. The _canary_ configuration enables experimental components and features that may be turned off in _production_. Each is served to 0.5% of AMP traffic. It is possible to opt into the **experimental** or **beta** channels via the `AMP_CANARY` cookie.
 
-- The ["Type: Release" GitHub issues](https://github.com/ampproject/amphtml/labels/Type%3A%20Release) for each release build will include a link to the specific [release page](https://github.com/ampproject/amphtml/releases) which lists the PRs in that release.
-  - The most recent "Type: Release" issue with `(Production)` in the title is the build currently in production.
-  - The "Type: Release" issue with `(Canary)` in the title tracks the current canary.
-- The [PR Use: In Canary](https://github.com/ampproject/amphtml/issues?utf8=%E2%9C%93&q=label%3A%22PR%20use%3A%20In%20Canary%22) and [PR Use: In Production](https://github.com/ampproject/amphtml/issues?utf8=%E2%9C%93&q=label%3A%22PR%20use%3A%20In%20Production%22) labels are added to PRs when they've made it into a canary/production build. There may be a delay between when the build is created and when it goes live.
+The **stable** release is built with the _production_ configuration and served to most AMP traffic. Since the **beta** release is also built from the _production_ configuration, it represents the exact build which will become **stable** the following week (ignoring the possibility of cherry-picks; see [Contributing Code](https://github.com/ampproject/amphtml/blob/master/contributing/lts-release.md)).
 
-## Cherry picks
+### Long-Term Stable (lts)
 
-We have a well-defined process for handling requests for changes to the canary release build or to the current production release build. These changes are known as "cherry picks".
+The **lts** release channel provides a previous **stable** build for four-week intervals. Every four weeks, the current **stable** release is promoted to **lts**. This is not recommended for all publishers building experiences with AMP, but is provided so that those performing a QA cycle may do so less often. Individual pages can explicitly opt into **lts** (see [**lts** Readme](https://github.com/ampproject/amphtml/blob/master/contributing/lts-release.md).
```

Comment:
Same nit as in the LTS doc: `less frequently` instead of `less often`.

Reply:
Copied over rewritted text from LTS doc here.